src/config/cloudtrail.json

```diff
@@ -29,27 +29,130 @@
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'DescribeDBSnapshots', &EmptyValue, &EmptyValue, @ )"
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'DescribeDBSubnetGroups', &EmptyValue, &EmptyValue, @ )"
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'DescribeEvents', &EmptyValue, &EmptyValue, @ )"
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'DescribeEventSubscriptions', &EmptyValue, &EmptyValue, @ )"
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'DescribeGlobalClusters', &EmptyValue, &EmptyValue, @ )"
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'DescribeInstallationMedia', &EmptyValue, &EmptyValue, @ )"
+        },
 
         {
-          "key": "dt.source_entity_type",
-          "description": "If a field given in condition_field exists and matches the condition value, use the value from result.",
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'DescribeOptionGroups', &EmptyValue, &EmptyValue, @ )"
+        },
 
-          "pattern": "[ { condition: `some other source of cloudTrail log event`, condition_field: log_content_parsed.eventSource, result: 'some other entity, e.g. CUSTOM_DEVICE'} ] | [?condition_field == condition] | [0] | result"
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'DescribeOrderableDBInstanceOptions', &EmptyValue, &EmptyValue, @ )"
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'DescribeRecommendationGroups', &EmptyValue, &EmptyValue, @ )"
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'ListTagsForResource', &(dt_meid_rds_v2( log_content_parsed.requestParameters.resourceName)), &EmptyValue, @ )"
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'RebootDBInstance', &dt_meid_rds_v2( format('arn:aws:rds:{}:{}:db:{}', region, account_id, log_content_parsed.requestParameters.dBInstanceIdentifier)), &EmptyValue, @ )"
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'DescribeEvents', &EmptyValue, &EmptyValue, @ )"
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "if( log_content_parsed.eventName == 'RebootDBInstance', &dt_meid_rds_v2( format('arn:aws:rds:{}:{}:db:{}', region, account_id, log_content_parsed.requestParameters.dBInstanceIdentifier)), &EmptyValue, @ )"
```

Comment:
1)I think that we should probably introduce "switch" instruction to jmespath.
2) One thing to check is where arguments evaluation happen. To avoid evaluating all arguments to if/switch this evaluation should preferably be done only after condition is true. But this might be asking for too much, right? 
Otherwise each if's arguments gets evaluated? Lot's of unnecessary calculations.

Reply:
1. I'm against the switch as mentioned on demo, at least the way I understand what you suggest (=big single rule with one switch for them all, line that cannot be broken or formatted). If you don't agree let's have a call.
2. As mentioned on demo, & makes it late resolved. If you have doubts, I can demo in on a call.

Reply:
1. Yes, big single rule, is it not possible to have this with multiple lines??
2. Great

Reply:
1. no, no line breaks within a string in json. Sad us during the hackaton.

Reply:
Oh right. 😞